services/web/server/src/simcore_service_webserver/diagnostics_healthcheck.py

```diff
@@ -111,7 +108,7 @@ def assert_healthy_app(app: web.Application) -> None:
                 max_delay,
                 max_delay_allowed,
             )
-            raise HealthError(msg)
+            raise HealthCheckFailed(msg)
 
     # CRITERIA 2: Mean latency of the last N request slower than 1 sec
```

Comment:
I'd prefer to have especially the maximum allowed __mean__ value not hardcoded as a magic number but settable via config. If this is the case, refer to the ENV-VAR that controls this in the comment instead of "1 sec"

Reply:
This was tuned after some experiments we did at a time.. I am open to change it when needed